### ISSUE TITLE
Observer for fraction excreted in bile in rats

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/PK-Sim#2773]
Create an observer for fraction excreted in bile in rats. Amount excreted should be an RHS parameter with RHS = rate of biliary transport.

**Comment by Yuri05** [comment]
Easy to create in MoBi

**Comment by PavelBal** [comment]
Yes, but not in a generic way ;)

**Comment by PavelBal** [comment]
Hmm actually no, I did not find a way to do it as observer, only as a parameter, because I need a RHS parameter...

**Comment by Yuri05** [comment]
> Hmm actually no, I did not find a way to do it as observer, only as a parameter, because I need a RHS parameter...

Yep, that was what I meant: activate "Create Process Rate" parameter for the biliaryclearance process and assign it to the RHS of a new parameter (amount excr) in MoBi

**Comment by PavelBal** [comment]
Ah yes, almost fortog about the "Create Process Rate" parameter.

**Comment by PavelBal** [comment]
But then the observer cannot be used in PK-Sim, right? Because "Create Process Rate" is not available in PK-Sim.
